src/utils/dates.js

```diff
@@ -13,7 +13,15 @@ export function getFutureDate(offset) {
 
 // transforms Firebase date object to JS date object
 export function transformToJSDate(date) {
```

Comment:
This function was refactored to return a new Date timestamp without the hours accounted for. The difference in hours would determine the order the items were sorted and the alphabetical sorting did not look accurate because sometimes an item would be a few hours apart. 

For example: Apples, Coffee, and Bananas were all purchased April 25, but because Bananas were added a little bit after Coffee the list would like Apples, Coffee, and Bananas. With the new fix, it will render Apples, Bananas, and Coffee if they were all purchased on the same day regardless of time.